tests/kernel/install_ltp.pm

```diff
@@ -47,6 +47,23 @@ sub add_we_repo_if_available {
 }
 
 sub install_runtime_dependencies {
+    # gettext-runtime-mini conflicts with sysstat, but it has grub2 dependencies.
```

Comment:
Initially I tried to just remove gettext-runtime-mini, but that will remove grub2 packages too. The reason I'm installing gettext-runtime is because sysstat requires it ("Problem: sysstat-10.2.1-10.12.1.x86_64 requires gettext...") and the mini version seems to not be enough.The zypper conflict installs gettext-runtime after uninstalling the mini version. I guess the comment doesn't explain it too well since I'm not explicitly removing the mini one. Sorry for the confusion. 

Also gettext-runtime-mini is what is installed by default on JeOS while on SLES its gettext-runtime so I do not know if there is a way to make it work with a different mini version, but I will look more into what sysstat actually requires from gettext.

I don't know if it helps with anything but here is a run with the sysstat conflict if you want to look into it: http://ccret.suse.cz/tests/964#step/install_ltp/67

Thank you,
Ciprian

Reply:
Thanks this helps explain. 

sysstat is nice to have, but you could disable it on JeOS, we call it after every test for debugging information and because it tests the performance counters, but it is not strictly necessary. However it might be just as easy to wrap the code you have already written in zypper_call.

Reply:
I updated it with zypper_call.
Here are the new runs: 
install: http://ccret.suse.cz/tests/997
crashme: http://ccret.suse.cz/tests/998
syscalls: http://ccret.suse.cz/tests/999